module/VuFind/src/VuFind/Search/EPF/Params.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace VuFind\Search\EPF;
+
+use VuFindSearch\ParamBag;
+
+class Params extends \VuFind\Search\Base\Params
+{
+
+    public function getBackendParameters()
+    {
+        $backendParams = new ParamBag();
+
+        // The documentation says that 'view' is optional, 
+        // but omitting it causes an error.
+        // https://connect.ebsco.com/s/article/Publication-Finder-API-Reference-Guide-Search
+        $view = $this->getEpfView();
+        $backendParams->set('view', $view);
+
+        return $backendParams;
+    }
+
+    public function getView()
+    {
+        $viewArr = explode('|', $this->view ?? '');
+        return $viewArr[0];
+    }
+
+    public function getEpfView() {
```

Comment:
I prefer to leave it.  EDS/Params.php has a slightly more complicated version of this method, which I simplified because the MVP is skipping the more advanced options/params logic from EDS.  But it's reasonable to think that may come later.  So I prefer to keep it parallel.